app/controllers/v2/content_items_controller.rb

```diff
@@ -24,6 +24,7 @@ def linkables
     def embedded
```

Comment:
another rename here, maybe it would be bigger change than I thought if we'd have to rename the endpoint 😬